net-misc/tigervnc/tigervnc-1.12.0-r5.ebuild

```diff
@@ -61,6 +63,8 @@ RDEPEND="${CDEPEND}
 	server? (
 		dev-lang/perl
 		sys-process/psmisc
+		!<x11-base/xorg-server-$(ver_cut 1-2 ${XSERVER_VERSION})
+		!>=x11-base/xorg-server-$(ver_cut 1 ${XSERVER_VERSION}).$((1 + $(ver_cut 2 ${XSERVER_VERSION})))
 	)"
```

Comment:
Is there an easier way to write this?
Something like ` !=x11-base/xorg-server-${XSERVER_VERSION%.*}*` but reversed

Reply:
I feel like what we really want is =...* still? The blocker is going to be confusing and saying "we can't be parallel installed with most versions" is a weird way of expressing "I really want version X". Like, isn't the effect the same here as before, just with a far more confusing error? You still can't install any version of xorg-server, right?

(Please tell me if missing something, I'm a bit tired!)

What does "wrong versions" mean? What happens if it's installed with versions that don't work?

Reply:
Nothing really bad happens, only the module will not work.

What I wanted to express was "either version ${XSERVER_VERSION} or no xorg-server at all".  If I leave it as it was, it will pull xorg-server on my headless machine.  It does look awkward, and it may be a bad idea.  I'm looking for feedback :)

Reply:
I have a new idea, this might be better.  I removed the blocker and added a metapackage instead.

So now tigervnc installs the xorg module, and users can install tigervnc-xorg-module that depends on xorg-server and pins it to the correct version.  Users that do not use the module can ignore it, and can also not install xorg-server at all.

The metapackage does what the USE flag was doing, but does not force a needless rebuild if added or removed.

Reply:
Nice solution!